src/Sylius/Behat/Context/Ui/Admin/ManagingProductsContext.php

```diff
@@ -977,4 +977,31 @@ private function resolveCurrentPage()
             $this->updateConfigurableProductPage,
         ]);
     }
+
+    /**
+     * @When I remove its price from channel :channel
+     */
+    public function iRemoveItsPriceFromChannel(ChannelInterface $channel)
```

Comment:
`public` functions should be placed before `private`. Also, do we have similar functions for `specifyPrice` and `checkChannel`? Maybe we can unify calls.